conda/cli/main_config.py

```diff
@@ -346,9 +353,121 @@ def print_config_item(key, value):
                 stdout_write(" ".join(("--add", key, repr(item))))
 
 
-def execute_config(args, parser):
+def _get_key(
+    key: str,
+    config: dict,
+    *,
+    json: dict[str, Any] = {},
+    warnings: list[str] = [],
+) -> None:
+    from ..base.context import context
+
+    key_parts = key.split(".")
+
+    if key_parts[0] not in context.list_parameters():
+        if context.json:
+            warnings.append(f"Unknown key: {key_parts[0]!r}")
+        else:
+            print(f"Unknown key: {key_parts[0]!r}", file=sys.stderr)
```

Comment:
I'm curious what the reason was for switching from `stderr_write` here?

Reply:
`stderr_write` is `getLogger("conda.stderr").info`

As mentioned in past logging discussions it is not clear when and why we should be using the logging infrastructure to print to the terminal instead of a `print` statement. This is all something I anticipate the output backend + logging refactors will resolve for us. Until that time I'm continuing to remove these in favor of simple `print` statements whenever possible.